docker-prune-cron/prune_docker_images.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -o errexit
+
+LOGFILE="$HOME/docker_prune.log"
+
+{
+  echo "Starting Docker system prune at $(date)"
+  sudo /usr/bin/docker system prune -f
+  echo "Pruned Docker objects successfully"
+
+  echo "Starting Docker volume prune at $(date)"
+  sudo /usr/bin/docker system prune --volumes -f
+  echo "Pruned Docker volumes successfully"
+  echo "Completed at $(date)"
```

Comment:
1. Do we need to specify full paths to docker binary?
2. Do we need to run it twice for each object separately?
3. Dates are better in ISO 8061 format is best
4. In scripts, prefer to use long format args (`--force` instead of `-f`)
```suggestion
  echo "Starting Docker prune at: $(date -u +"%Y-%m-%dT%H:%M:%SZ")"
  sudo docker system prune --volumes --force
  echo "Completed Docker prune successfully at: $(date -u +"%Y-%m-%dT%H:%M:%SZ")"
```

Reply:
> Do we need to specify full paths to docker binary?
My idea was just for consistency purposes, such that the script uses docker binary located at that  specific path. But I guess it's overkill, so I'll fix it back.
> Do we need to run it twice for each object separately?
Yes, I have seen the mistake. Thank you.